external_status_checks.go

```diff
@@ -88,3 +88,99 @@ func (s *ExternalStatusChecksService) ListProjectStatusChecks(pid interface{}, o
 
 	return pscs, resp, err
 }
+
+type SetExternalStatusCheckStatusOptions struct {
+	SHA                   *string `url:"sha" json:"sha"`
+	ExternalStatusCheckID *int    `url:"external_status_check_id" json:"external_status_check_id"`
+	Status                *string `url:"status,omitempty" json:"status,omitempty"`
+}
```

Comment:
Please all a header (comment) and add missing `omitempty` tag options (should be on all fields).